### ISSUE TITLE
Speech Keyword Spotting with Brain.js

### ISSUE THREAD

**Issue by cycomp** [BrainJS/brain.js#558]
I am new to neural networks and have followed the scrimba tutorials. I want to do keyword spotting using the Speech Commands Dataset (https://www.tensorflow.org/datasets/catalog/speech_commands)
Specifically I want to identify the words "on" and "off", so I will have 3 outputs, "on", "off", "unknown word".

The inputs are from 1 second long signed 16bit WAV files, I've converted them into a javascript Int16Array with 16,000 values. I understand that an LSTM network is best for speech recognition, but with the current version of brain.js that makes an input for every distinct value, which is not reasonable.

I think what is needed is a convolutional approach, with a feature detector identifying the "o" sound, the "n" sound and the "ff" sounds, and then a memory layer which determines whether they have been "heard" in the correct order.

Does anyone have any pointers of how to go about this?

**Comment by robertleeplummerjr** [comment]
>I understand that an LSTM network is best for speech recognition, but with the current version of brain.js that makes an input for every distinct value, which is not reasonable.

If you could normalize the training data to numbers, this may be feasible with `brain.recurrent.LSTMTimeStep`.  I've not seen anyone do this with brain.js thus far, but I'd be especially interested to see it.
